cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -64,6 +64,12 @@ def __init__(self, original_table, users_table, dynamodbclient):
 
     def add_user_info(self, user_info):
 
+        # get ttl_expiration from test users if they exist
+        try:
+            ttl_expiration = {"N":str(user_info['ttl_expiration'])}
+        except:
+            ttl_expiration = {"S":""}
```

Comment:
The mixed attribute type here makes me a bit nervous.  What was the rationale?

Reply:
The ttl functionality only triggers if the dynamo table attribute is a Number type. All the testing elements should have this ttl_expiration attribute (so they won't raise the exception) and be a Number type in AWS. I wanted to make sure no non-testing items would have ttl triggered by accident, so I just set the ttl_expiration to an empty String type. If it was an empty Number type, I wasn't sure if it would cause any deletions, so to be safe I just changed it to a String type.